src/seedu/addressbook/Main.java

```diff
@@ -16,6 +16,7 @@
 import seedu.addressbook.ui.TextUi;
 
 
+
```

Comment:
This change is not relevant right?

Reply:
I am not sure why there is an extra line in the code. Do i need to delete this? Or how do I do about this?

Reply:
You can delete the extra line. On sourcetree you can review the changes you made to the code before staging them :)

Reply:
I saw 2 other blank lines added on other files as well, maybe it's a setting in your IDE?

Reply:
which specific file is that?

Reply:
Another one is in `src/seedu/addressbook/data/AddressBook.java` as @javenseow pointed out. There was one more but looks like it's already fixed.